tests/unit/gapic/servicemanagement_v1/test_service_manager.py

```diff
@@ -25,7 +25,7 @@
 from proto.marshal.rules.dates import DurationRule, TimestampRule
 
 from google import auth
-from google.api import auth_pb2 as ga_auth  # type: ignore
+from google.api import auth_pb2 as auth  # type: ignore
```

Comment:
This change should be reverted. The issue has been patched in generator version [0.43.2](https://github.com/googleapis/gapic-generator-python/pull/823).
```suggestion
from google.api import auth_pb2 as ga_auth  # type: ignore
```